src/js/media-play-button.ts

```diff
@@ -53,13 +87,32 @@ class MediaPlayButton extends MediaChromeButton {
     ];
   }
 
+  tooltip: HTMLElement;
+
   constructor(options = {}) {
     super({ slotTemplate, ...options });
+    this.tooltip = this.shadowRoot.querySelector('#tooltip');
   }
 
   connectedCallback(): void {
     updateAriaLabel(this);
     super.connectedCallback();
+    this.addEventListener(
```

Comment:
This feels like generic tooltip logic that should go in media-chrome-button, in case that's not already the plan.

Reply:
That is the plan! just working within this control while the logic for everything is worked out. Still some refactoring to do